python/desc/imsim/atmPSF.py

```diff
@@ -8,6 +8,21 @@
 import galsim
 from lsst.sims.GalSimInterface import PSFbase
 
+from .optical_system import OpticalZernikes, mock_deviations
+
+class OptWF(object):
```

Comment:
@jmeyers314 I included this commit in preparation for merging this branch into master, but in running the code, I got this error:
```
Traceback (most recent call last):                                                
  File "/global/cscratch1/sd/jchiang8/imsim_pipeline/imSim/bin/imsim.py", line 190, in <module>
    main()
  File "/global/cscratch1/sd/jchiang8/imsim_pipeline/imSim/bin/imsim.py", line 169, in main
    gs_interpreter.drawObject(gs_obj)
  File "/global/cscratch1/sd/jchiang8/imsim_pipeline/sims_GalSimInterface/python/lsst/sims/GalSimInterface/galSimInterpreter.py", line 621, in drawObject
    centeredObj = self.findAllDetectors(gsObject)
  File "/global/cscratch1/sd/jchiang8/imsim_pipeline/sims_GalSimInterface/python/lsst/sims/GalSimInterface/galSimInterpreter.py", line 179, in findAllDetectors
    centeredObj = self.createCenteredObject(gsObject)
  File "/global/cscratch1/sd/jchiang8/imsim_pipeline/sims_GalSimInterface/python/lsst/sims/GalSimInterface/galSimInterpreter.py", line 474, in createCenteredObject
    centeredObj = self.drawPointSource(gsObject, psf=psf)
  File "/global/cscratch1/sd/jchiang8/imsim_pipeline/sims_GalSimInterface/python/lsst/sims/GalSimInterface/galSimInterpreter.py", line 385, in drawPointSource
    return psf.applyPSF(xPupil=gsObject.xPupilArcsec, yPupil=gsObject.yPupilArcsec)
  File "/global/cscratch1/sd/jchiang8/imsim_pipeline/sims_GalSimInterface/python/lsst/sims/GalSimInterface/galSimPSF.py", line 69, in applyPSF
    psf = self._getPSF(xPupil=xPupil, yPupil=yPupil, **kwargs)
  File "/global/cscratch1/sd/jchiang8/imsim_pipeline/imSim/python/desc/imsim/atmPSF.py", line 150, in _getPSF
    exptime=self.exptime)
  File "/global/cscratch1/sd/jchiang8/imSim_prs/galsim_local/python/galsim/phase_psf.py", line 939, in makePSF
    return PhaseScreenPSF(self, lam, theta=theta, **kwargs)
  File "/global/cscratch1/sd/jchiang8/imSim_prs/galsim_local/python/galsim/phase_psf.py", line 1174, in __init__
    gsparams=self._gsparams)
  File "/global/cscratch1/sd/jchiang8/imSim_prs/galsim_local/python/galsim/phase_psf.py", line 971, in _stepK
    return np.sum([layer._stepK(**kwargs)**(-5./3) for layer in self])**(-3./5)
  File "/global/cscratch1/sd/jchiang8/imSim_prs/galsim_local/python/galsim/phase_psf.py", line 971, in <listcomp>
    return np.sum([layer._stepK(**kwargs)**(-5./3) for layer in self])**(-3./5)
AttributeError: 'OptWF' object has no attribute '_stepK'
```
I'm using GaSim v1.6.0.  It looks like this class needs the `_stepK` function.  Could you have a look please?

Reply:
It seems that adding this function to the `OptWF` class is all that's needed:
```
    def _stepK(self, **kwds):
        lam = kwds['lam']
        diam = kwds['diam']
        obscuration = kwds['obscuration']
        gsparams = kwds['gsparams']
        obj = galsim.Airy(lam=lam, diam=diam, obscuration=obscuration, gsparams=gsparams)
        return obj.stepk
```
@jmeyers314 Can you confirm please?

Reply:
@jchiang87 , yes, that looks right.  Just pushed effectively that.

Reply:
Great! Thanks, @jmeyers314 .